### DISCUSSION TITLE
Debugger for Cefsharp not working after upgrading it to 115.4.0 version, earlier with 83.0.1 version it was working fine

### DISCUSSION THREAD

**Discussion by AanchalTewary** [cefsharp/CefSharp#5013] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

115.4.0

### Operating System

Windows 10

### Architecture

x86

### .Net Version

.Net 4.8

### Implementation

WPF

### Reproduction Steps

Debugger for Cefsharp is not working after upgrading it to cefsharp version 115.4.0 while it was working fine for version 83.0.1
It just shows Inpectable Web Contents and shows nothing below it although the web windows are open.

### Expected behavior

It should list all the web windows and allow debugging.

### Actual behavior

On opening https://localhost:9999 it shows just Inspectable Web Contents and nothing under it. No errors in the console as well. Please advise.

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

### Other information

Please advise how to look for the root cause and resolve the issue

**Comment by amaitland** [comment]
> What version are you using? Please only open an issue if you can reproduce the problem with version 130.1.90 or later.

As per https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

> On opening https://localhost:9999 it shows just Inspectable Web Contents and nothing under it. No errors in the console as well. Please advise.

From memory `Chromium` added additional security constrains. There should be an error in the log file telling you what is required.

`remote-allow-origins`

This is no longer supported in the current version, see https://github.com/chromiumembedded/cef/issues/3740#issuecomment-2231301000
